partiql-ast/src/ast.rs

```diff
@@ -315,7 +250,6 @@ pub enum Lit {
     NationalCharStringLit(String),
     BitStringLit(String),
     HexStringLit(String),
-    DateTimeLit(DateTimeLit),
```

Comment:
Why are `DateTimeLit` and `Param` on L348 being removed?

Reply:
Nothing ever parsed to either of them.

Date/Time literals are parsed into `TypedLit`s at current.

Reply:
Ah, gotcha